Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1,17 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Core.Definitions.Devices;
-using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
-using Microsoft.MixedReality.Toolkit.Core.EventDatum.Input;
-using Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem;
-using Microsoft.MixedReality.Toolkit.SDK.Input.Handlers;
-using Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem.Handlers;
-using Microsoft.MixedReality.Toolkit.Core.Services;
+using Microsoft.MixedReality.Toolkit.Input;
 using System.Collections.Generic;
 using UnityEngine;
+using UPhysics = UnityEngine.Physics;
```

Comment:
>UPhysics [](start = 6, length = 8)

nit: Might be more clear to call UnityPhysics, but I guess that isn't much better than UnityEngine.Physics

Reply:
I could also put this type inside the namespace block:
using Physics = UnityEngine.Physics;

I  am fine either way

---
In reply to: [267119045](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267119045) [](ancestors = 267119045)

Reply:
I think there are some other examples for Boundary where we have duplicate namespace ending words. It may make more sense to keep UPhysics or move to UnityPhysics over Physics for these sorts of cases.

---
In reply to: [267119750](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267119750) [](ancestors = 267119750,267119045)